test.rb

```diff
@@ -0,0 +1,24 @@
+require "./lib/api.rb"
+
+require 'pp'
+
+api_token = ENV['REPLICATED_API_TOKEN']
+app_id = ENV['REPLICATED_APP_ID']
```

Comment:
Doesn't look like you're using this, it's being overwritten in the map below.

Reply:
True